examples/visualization/plot_sensor_noise_level.py

```diff
@@ -29,3 +23,8 @@
 # We can plot the absolute noise levels:
 raw_erm.plot_psd(tmax=10., average=True, spatial_colors=False,
                  dB=False, xscale='log')
+###############################################################################
+# References
+# ----------
+#
+# .. footbibliography::
```

Comment:
After doing a quick check of the other tutorials, there is usually no blank line here

```suggestion
# .. footbibliography::
```